### ISSUE TITLE
Invalid support for exclusiveMinimum/exclusiveMaximum

### ISSUE THREAD

**Issue by mclate** [Redocly/redoc#1379]
Looks like redoc doesn't handle `exclusiveMinimum` and `exclusiveMaximum` in accordance with current JSON schema draft. Based on the lines below, in order to display numeric constraints, it is required to have `minimum`/`maximum` set in the schema, however, they seem to be mutually exclusive with `exclusiveMinimum` and `exclusiveMaximum`, meaning that exclusive values will never be evaluated and will never be displayed. 
https://github.com/Redocly/redoc/blob/f3e8ab4f8e5522c9ea1ddedb143e23c7d62f5807/src/utils/openapi.ts#L427-L439

After some digging, i figured that in JSON schema draft 4 `exclusiveMinimum` and `exclusiveMaximum` were boolean fields (making code above totally valid) however, i can see that current draft version of JSON schema is 7, making version 4 quite old now... In latest draft exclusive values are numeric now.

Side note: JSON schema draft 7 says that

> While you can specify both of minimum and exclusiveMinimum or both of maximum and exclusiveMaximum, it doesn’t really make sense to do so.

**Comment by pjungermann** [comment]
Actually, there is already an even newer version 2019-09 (fka. draft 8).

OAS v3 is an extended subset of draft 5 (from 2016-10; aka. "Wright Draft 00") which is in essence draft 4 (a rewrite of draft 4 as preparation for draft 6)
- http://spec.openapis.org/oas/v3.0.3#schema-object
- http://json-schema.org/specification-links.html#draft-5
- https://json-schema.org/draft-06/json-schema-release-notes.html#q-what-happened-to-draft-05

Therefore, in OAS v3 `exclusiveMinimum` and `exclusiveMaximum` are still boolean fields

**Comment by adamaltman** [comment]
Yes, and OAS3.1 will support 2019-09: https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.1.0.md#data-types

**Comment by AlexVarchuk** [comment]
It was fixed. Please reopen this issue if you have some questions.
